lib/reporters/cli/index.js

```diff
@@ -108,11 +109,20 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
             }
         }
 
+        if (run.transfers.responseTotal > 10485760) {
```

Comment:
Since the output size conditional only applies to output printing, so lets move this if conditional inside of the above if conditional.